docs/usage.md

```diff
@@ -0,0 +1,107 @@
+# Usage
+
+There are two main ways to use nion from React: as a _hook_, or as a _HOC_ (higher-order component). The HOC usage itself comes in two forms: modern HOC and legacy decorators.
+
+## As a hook
+
+When writing new code, the recommended way to use nion is the `useNion` [hook](https://reactjs.org/docs/hooks-intro.html). Note that the `dataKey` in the declaration is required.
+
+```js
+import { useNion } from '@nion/nion'
+
+export const UserContainer = () => {
+    const [currentUser, actions, request] = useNion({
```

Comment:
For those used to the decorator syntax, they know that if the React DOM looks like this:
```
<Provider>
    <ParentContainer>
        <ChildComponent/>
    </ParentContainer>
</Provider>
```

then 
```js
// ParentContainer/index.jsx
@nion({
    currentUser: {
        endpoint: 'https://patreon.com/api/current_user'
    }
})
class ParentContainer extends React.Component {
    ...
}

// ChildComponent/index.jsx
@nion('currentUser')
class ChildComponent extends React.Component {
    ...
}
```

Because nion dataz are stored in our Redux store, is the equivalent in hooks to just do
`useNion({dataKey: 'currentUser'})` for `ChildComponent` above?

Reply:
Without additional research, I don't have a proper answer for this great question, but I will note that it falls under Item 4 of #109 ("Reorganize the three declaration forms — function-returning-object, object, string — as a cascade of syntactic sugar"), and so it will definitely be addressed, but not in this PR.

Summarizing this issue: how does `dataKey` abbreviation work, and how do we explain it intuitively yet thoroughly?

Reply:
The hook takes a string: `useNion('currentUser')` works just fine.